### PR TITLE
Remove kobzol vacation status

### PR DESCRIPTION
**Pull request by Kobzol**
Conflicts with https://github.com/rust-lang/rust/pull/129843, so should be merged after it.

### CODE REVIEW

File: triagebot.toml [hunk 1]
```diff
@@ -913,7 +913,7 @@ cc = ["@kobzol"]
 [assign]
 warn_non_default_branch = true
 contributing_url = "https://rustc-dev-guide.rust-lang.org/getting-started.html"
-users_on_vacation = ["jyn514", "jhpratt", "oli-obk", "kobzol", "joboet"]
+users_on_vacation = ["jyn514", "jhpratt", "oli-obk", "joboet"]
```
Comment (workingjubilee) [inline triagebot.toml:916]: This should remain a multiple-line array.

**Comment by Kobzol** [comment]
Rebased.

@bors r=lqd

**Pull request merged**